src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1070,6 +1070,8 @@ namespace NServiceBus.AutomaticSubscriptions.Config
 {
     public class AutoSubscribeSettings
     {
+        [System.ObsoleteAttribute("Encourages bad practices. IBusSession.Subscribe should be explicitly used. Will b" +
```

Comment:
Encourages bad practices. ? This message isn't very clear. Can you add more context?